src/modules/users/users.module.ts

```diff
@@ -4,9 +4,11 @@ import { forwardRef, Module } from '@nestjs/common';
 import { UsersController } from './controllers/users.controller';
 import { UsersService } from './services/users.service';
 
+import { CampaignMonitorModule } from '~modules/campaign-monitor';
+
 @Module({
 	controllers: [UsersController],
-	imports: [forwardRef(() => DataModule)],
+	imports: [forwardRef(() => DataModule), CampaignMonitorModule],
```

Comment:
ensure you check the unit tests for this module still work
if not add ehte CampaignMonitorModule to the test mocks

eg, try to run:
```
npm run test
```